release-content/0.15/release-notes/14630_A_Curve_trait_for_general_interoperation__Part_I.md

```diff
@@ -1,27 +1,35 @@
 <!-- A Curve trait for general interoperation — Part I -->
 <!-- https://github.com/bevyengine/bevy/pull/14630 -->
 
-The new `Curve` trait provides a shared interface for curves.
+The new [`Curve<T>`] trait provides a shared interface for curves, describing how values of type `T` change as we vary a `f32` parameter `t` over some domain.
 
-`Curve<T>` defines a value of some type `T`
-parametrized by a nonempty closed interval of real numbers. That parameter could,
-for example, represent time, in which case a `Curve<T>` is thought of as a value
-of type `T` that changes over time, as in animation. The parameter
-could also represent something like distance or displacement, as in gradients and 
-spatial curves.
+What's changing, and the domain that it's changing *over* are both incredibly flexible.
+You might choose to set `T` to store colors, creating a powerful abstraction for [color gradients](https://docs.rs/bevy/0.15.0/bevy/color/struct.ColorCurve.html), position, or the experience your player needs to level up.
```

Comment:
Not sure if experience per level is a great example since levels are a discrete thing, so an array of values might be more suitable.

A different game progression metric comes to mind: Something like damage parametrised by the amount of experience the player has.
(While this could also be discrete, it does make sense to view it as a curve that would make sense even for intermediate values.)